### PR TITLE
Fedora-support

### LINKED ISSUE #642: [Fedora] Create a test for setup script
Verify tmt flow is working on CloudBase Fedora 41

### PR DESCRIPTION
**Pull request by Yarboa**
resolve #642 

Prepare Fedora image for testing
Changing the copr repositories to more genereric way
README explains how to tests copr PR rpm on local vm
For cases without /etc/os-release added distro context
to env-variable convert in main plan

### CODE REVIEW

File: tests/e2e/set-ffi-env-e2e [hunk 1]
```diff
@@ -156,13 +157,18 @@ install_qm_rpms() {
...
+  if [[ "$release_id" == "centos" ]]; then
+    release_id=epel
+  fi
 
   if [[ -n "${USE_QM_COPR}" ]]; then
       if [[ "${USE_QM_COPR}" != "release" ]]; then
-          dnf copr enable -y @centos-automotive-sig/bluechi-snapshot centos-stream-9
-          dnf copr enable -y "${USE_QM_COPR}" centos-stream-9
-      elif [[ "${USE_QM_COPR}" == "release" && "${release_id}" =~ "centos" ]]; then
+          dnf copr enable -y @centos-automotive-sig/bluechi-snapshot "$release_id-$version_id-$(arch)"
```
Comment (dougsland) [inline tests/e2e/set-ffi-env-e2e:169]: Question, isn't now part of .packit.yaml? (do we need it here too)?  Maybe we can remove in one place?

Comment (Yarboa) [reply tests/e2e/set-ffi-env-e2e:169]: this one, for installing PR copr rpm for build, 
There are many options for manual tests and debug
see the command, https://github.com/containers/qm/pull/652/files#diff-fd76bc4afd6e791b352e18d8bcda536f540475342be7ef1f72737960369d3343R271

Comment (dougsland) [reply tests/e2e/set-ffi-env-e2e:169]: No, I meant, in theory this enable should be automatic these days with .packit.yaml. Right?

Comment (Yarboa) [reply tests/e2e/set-ffi-env-e2e:169]: Yes, but i would like to propose it in a different PR once ros or libkrun fedora tests to be introduced

Comment (dougsland) [reply tests/e2e/set-ffi-env-e2e:169]: @Yarboa so should we merge this one and later you address the duplication?

File: tests/e2e/README.md [hunk 2]
```diff
@@ -258,3 +261,12 @@ Running FFI tests connecting c9s host, sets different tests environment from the
 tmt -c scenario=ffi run -a prepare  provision --how connect \
     -u root -p ${PASSWORD} -P {PORT} -g localhost plan -n /plans/e2e/ffi
 ```
+
+##### Running manual tests with PACKIT copr packages
+
+Assume debug of PR rpm needed to run locally,
+Use context `-c run=manual` with environment variable `-e PACKIT_COPR_PROJECT=<packir-qm-copr>`
```
Comment (nforro) [inline tests/e2e/README.md:268]:
```suggestion
Use context `-c run=manual` with environment variable `-e PACKIT_COPR_PROJECT=<packit-qm-copr>`
```

**Pull request merged**